packages/maps/src/components/marker-cluster/MapMarkerClusterView.tsx

```diff
@@ -0,0 +1,62 @@
+import React from "react";
+import { ViewStyle, StyleProp, Text, View } from "react-native";
+import { MapMarkerClusterContext } from "./MapMarkerClusterContext";
+import { withTheme } from "@draftbit/ui";
+
+interface MapMarkerClusterViewProps {
+  zoomOnPress?: boolean;
+  onPress?: (latitude: number, longitude: number) => void;
+  renderItem?: ({ markerCount }: { markerCount: number }) => JSX.Element;
+  style?: StyleProp<ViewStyle>;
+}
+
+/**
+ * Overrides the default cluster component that is rendered when a group of markers are clustered together
+ *
+ * Placed inside a MapMarkerCluster when manually creating clusters
+ * OR inside a MapView when relying on automatic clustering based on distance
+ */
+const MapMarkerClusterView: React.FC<MapMarkerClusterViewProps> = ({
+  renderItem,
+  style,
+}) => {
+  const { markerCount } = React.useContext(MapMarkerClusterContext);
+
+  return (
+    <View style={style}>{renderItem?.({ markerCount: markerCount || 0 })}</View>
```

Comment:
Not really, my thought process is in case the user sets some styling (size, margins, etc) but did not add any children yet that it shows the applied changes.